website/src/components/HomepageFeatures/index.js

```diff
@@ -70,13 +70,13 @@ const FeatureList = [
 
 function Feature({ Svg, title, description }) {
   return (
-    <div className={clsx("col col--4")}>
-      <div className="text--center">
+    <div className={clsx("col col--6 padding-vert--md", styles.feature)}>
+      <div className={styles.svgContainer}>
         <Svg className={styles.featureSvg} role="img" />
       </div>
-      <div className="text--center padding-horiz--md">
-        <h3>{title}</h3>
-        <p>{description}</p>
+      <div className={styles.featureDetails}>
+        <h2 className={styles.featureTitle}>{title}</h2>
+        <h3 className={styles.featureInfo}>{description}</h3>
```

Comment:
`<h3>` isn't semantically the right tag to use here since there's no text under the heading. Should just use `<p>` but with increased font size or similar.